src/sub/tests/conftest.py

```diff
@@ -37,6 +37,13 @@ class MockCustomer:
     def properties(self, cls):
         return [i for i in cls.__dict__.keys() if i[:1] != "_"]
 
+    def __contains__(self, key, default=None):
+        properties = self.properties(MockCustomer)
+        if key in properties:
+            return key
+        else:
+            return default
+
     def get(self, key, default=None):
```

Comment:
No, the reason for the `get` calls is because if a value is not there and you just `""` then you get a KeyError, `.get` returns back None as a default if the key does not exist, which is the preferred behavior.